client/dashboard/index.js

```diff
@@ -26,22 +26,19 @@ const CustomizableDashboard = lazy( () =>
 	import( /* webpackChunkName: "customizable-dashboard" */ './customizable' )
 );
 
-const ProfileWizard = lazy( () =>
-	import( /* webpackChunkName: "profile-wizard" */ '../profile-wizard' )
-);
-
 class Dashboard extends Component {
 	render() {
 		const { path, profileItems, query } = this.props;
+		const { completed: profileCompleted, skipped: profileSkipped, step: profilerStep } = profileItems;
 		if (
 			isOnboardingEnabled() &&
-			! profileItems.completed &&
+			! profileCompleted &&
+			! profileSkipped &&
 			! window.wcAdminFeatures.homescreen
 		) {
-			return (
-				<Suspense fallback={ <Spinner /> }>
-					<ProfileWizard query={ query } />
-				</Suspense>
+			const lastStep = profilerStep ? `&step=${ profilerStep }` : '';
+			window.location = getAdminLink(
+				`admin.php?page=wc-admin&path=/profiler${ lastStep }`
```

Comment:
I think using React Router to change route can avoid loading the page twice. Here is an example:

https://github.com/woocommerce/woocommerce-admin/blob/e09675584236dceaa1f91fcb64f11b69ad52f9cc/client/layout/controller.js#L294

Reply:
Hi @psealock, great idea!
I changed it in the commit ebe323c